src/executor/stack/mod.rs

```diff
@@ -17,6 +17,26 @@ use ethereum::Log;
 use primitive_types::{H160, H256, U256};
 use sha3::{Digest, Keccak256};
 
+macro_rules! emit_exit {
```

Comment:
```suggestion
macro_rules! emit_and_return_exit {
```

Reply:
Well it doesn't actually contain the `return` part, the macro itself is an expression returning the parameters provided to it. In multiple places you can see `return emit_exit!(...)`.
If you still think this would be a better name I'll change it :)